src/probnum/diffeq/ode/ivp.py

```diff
@@ -6,196 +6,7 @@
-
-    Parameters
-    ----------
-    timespan : (float, float)
-        Time span of IVP.
-    initrv : RandomVariable,
-        RandomVariable that  describes the belief over the initial
-        value. Usually its distribution is Constant (noise-free)
-        or Normal (noisy). To replicate "classical" initial values
-        use the Constant distribution.
-    params : (float, float, float, float), optional
-        Parameters :math:`(a, b, c, d)` for the logistic IVP.
-        Default is :math:`(a, b, c, d)=(0.0, 0.08, 0.07, 1.25)`.
-
-    Returns
-    -------
-    IVP
-        IVP object describing the logistic IVP with the prescribed
-        configuration.
-    """
-
-    def rhs(t, y):
-        return fhn_rhs(t, y, params)
-
-    def jac(t, y):
-        return fhn_jac(t, y, params)
-
-    return IVP(timespan, initrv, rhs, jac)
-
-
-def fhn_rhs(t, y, params):
-    """RHS for FitzHugh-Nagumo model."""
-    y1, y2 = y
-    a, b, c, d = params
-    return np.array([y1 - y1 ** 3.0 / 3.0 - y2 + a, (y1 + b - c * y2) / d])
-
-
-def fhn_jac(t, y, params):
-    """Jacobian for FitzHugh-Nagumo model."""
-    y1, y2 = y
-    a, b, c, d = params
-    return np.array([[1.0 - y1 ** 2.0, -1.0], [1.0 / d, -c / d]])
-
-
-def lotkavolterra(timespan, initrv, params=(0.5, 0.05, 0.5, 0.05)):
-    """Initial value problem (IVP) based on the Lotka-Volterra model.
-
-    The Lotka-Volterra (LV) model is defined through
-
-    .. math:: f(t, y) =
-        \\begin{pmatrix} a y_1 - by_1y_2 \\\\ -c y_2 + d y_1 y_2
-        \\end{pmatrix}
-
-    for some parameters :math:`(a, b, c, d)`.
-    Default is :math:`(a, b)=(0.5, 0.05, 0.5, 0.05)`.
-    This implementation includes the Jacobian :math:`J_f` of :math:`f`.
-
-    Parameters
-    ----------
-    timespan : (float, float)
-        Time span of IVP.
-    initrv : RandomVariable,
-        RandomVariable that  describes the belief over the initial
-        value. Usually its distribution is Constant (noise-free)
-        or Normal (noisy). To replicate "classical" initial values
-        use the Constant distribution.
-    params : (float, float, float, float), optional
-        Parameters :math:`(a, b, c, d)` for the logistic IVP.
-        Default is :math:`(a, b, c, d)=(0.5, 0.05, 0.5, 0.05)`.
-
-    Returns
-    -------
-    IVP
-        IVP object describing the logistic IVP with the prescribed
-        configuration.
-    """
-
-    def rhs(t, y):
-        return lv_rhs(t, y, params)
-
-    def jac(t, y):
-        return lv_jac(t, y, params)
-
-    return IVP(timespan, initrv, rhs, jac)
-
-
-def lv_rhs(t, y, params):
-    """RHS for Lotka-Volterra."""
-    a, b, c, d = params
-    y1, y2 = y
-    return np.array([a * y1 - b * y1 * y2, -c * y2 + d * y1 * y2])
-
-
-def lv_jac(t, y, params):
-    """Jacobian for Lotka-Volterra."""
-    a, b, c, d = params
-    y1, y2 = y
-    return np.array([[a - b * y2, -b * y1], [d * y2, -c + d * y1]])
+__all__ = ["IVP"]
```

Comment:
same here. Where do we want the __all__ things? Is there something in the styleguide?

Reply:
> Use __all__ = [...] in __init__.py files to fix the order in which the methods are visible in the documentation. This also avoids importing unnecessary functions via import statements from ... import *.

From [here](https://github.com/probabilistic-numerics/probnum/blob/master/STYLEGUIDE.md). It is probably not all that important if you want to expose all the functionality from the module, but as it is made explicit in the styleguide I thought I'd better include it

Reply:
I see... though ode is not a real module in the sense that all its functions are pulled up to the diffeq namespace anyway. To me it is not clear whether this means that it needs its own __all__ or not... I am fine with keeping it until then, though especially here it seems a bit redundant...